### ISSUE TITLE
expose constructor for `KyberPreKeyRecord`

### ISSUE THREAD

**Issue by TheTechZone** [TheTechZone/signal-protocol.py#10]
`libsignal` offers pretty nice primitives for handling generating keys along with associated data (such as signatures).

However, `KyberPreKeyRecord` unlike other state types does not expose a constructor, only a static `generate()` function. The current workaround is to create a record by deserializing a `SignedPreKeyRecordStructure` protobuf. 

```python3
def make_kyber_record(key_id: int, ts: int, kp: KemKeyPair, signature: bytes) -> KyberPreKeyRecord:
    record = SignedPreKeyRecordStructure()
    record.id = key_id
    record.public_key = kp.get_public().serialize()
    record.private_key = kp.get_private().serialize()
    record.signature = signature
    record.timestamp = ts
    return KyberPreKeyRecord.deserialize(record.SerializeToString())
```

This works but is quite clunky and error prone. Alternatively we could just store the serialized bytestring but that would limit further possibilities.

**Comment by TheTechZone** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [KyberPreKeyRecord](https://trello.com/c/fT3TGHhB/18-kyberprekeyrecord)

**Comment by TheTechZone** [comment]
addressed by [a9431de](https://github.com/TheTechZone/signal-protocol.py/commit/a9431de1d9c0fe4ffed988b7d54c230fca5a8e9f)
